File: solutions/javascript/is-anagram.js
var isAnagram = function(stringOne, stringTwo) {
  var stringOneArr = stringOne.split('');
  var stringTwoArr = stringTwo.split('');

  stringOneArr.sort();
  stringTwoArr.sort();

  for (var i = 0; i < stringOneArr.length; i++) {
    if (stringOneArr[i] !== stringTwoArr[i]) return false;
  }

  return true;
}

console.log(isAnagram('abcd', 'dbca')) // --> true
console.log(isAnagram('abcd', 'aabb')) // --> false

File: solutions/python/is-anagram.py
def is_anagram(string_one, string_two):
    sorted_string_one = sorted(string_one)
    sorted_string_two = sorted(string_two)

    for char_one, char_two in zip(sorted_string_one, sorted_string_two):
        if char_one != char_two:
            return False

    return True

print(is_anagram('abcd', 'dbca')) # --> True
print(is_anagram('abcd', 'aabb')) # --> False
